blocks/library/quote/block.scss

```diff
@@ -17,17 +17,18 @@
 			top: 0;
 		}
 	}
-}
 
-.editor-visual-editor .blocks-quote-style-1 {
-	border-left: 4px solid $black;
-	padding-left: 1em;
-}
 
-.editor-visual-editor .blocks-quote-style-2 {
-	padding: 0 1em;
-	& > .blocks-editable p {
-		font-size: 24px;
-		font-style: italic;
+	&.blocks-quote-style-1 {
+		border-left: 4px solid $black;
+		padding-left: 1em;
+	}
+
+	&.blocks-quote-style-2 {
+		padding: 0 1em;
+		& > .blocks-editable p {
```

Comment:
This was an error I should move this back to `style.scss` and use `> p` instead for frontend.